controllers/admin/AdminSuppliersController.php

```diff
@@ -452,6 +452,10 @@ public function postProcess()
                 return;
             }
 
+            /** @var array $addressFields - The Supplier address fields */
+            $addressFields = array('alias', 'lastname', 'firstname', 'address1', 'address2', 'postcode', 'phone',
```

Comment:
RIP all overrides which adds new fields for Manufacturers and Suppliers addresses :)